Sources/Basic/Path.swift

```diff
@@ -503,11 +503,11 @@ extension AbsolutePath {
         return self.components.starts(with: other.components)
     }
 
-    /// Checks the path and throws a `PathValidationError` if the path is invalid.
-    public static func validate(pathString path: String) throws {
+    /// Checks the path and throws a `PathValidationError` if the path is invalid absolute path.
+    public static func validated(pathString path: String) throws -> String {
         switch path.first {
         case "/":
-            return
+            return path
```

Comment:
At this point, we can just return AbsolutePath. Maybe we should add a new throwing init that uses this method.

```swift
init(validating path: String) throws
```